src/domain/banner/bannerUtils.ts

```diff
@@ -0,0 +1,110 @@
+import { BannerPage, Maybe } from '../../generated/graphql';
```

Comment:
naming of the file is not in line with the other constant files. Not saying this worse, might be better actually but should we change the naming of the files in all places?

Reply:
I would rather rename them all than have multiple files with the same name. At least in intelliJ it makes searching by filenames easier

Reply:
vscode nicely shows the path for all the found files so having same names is not necessarily problem there but the case might be different for IntelliJ
<img width="634" alt="Screenshot 2020-12-23 at 8 44 00" src="https://user-images.githubusercontent.com/15219142/102967640-0f5a7a80-44fb-11eb-943b-5bc3ad7128bc.png">

Reply:
<img width="647" alt="Screenshot 2020-12-23 at 8 46 39" src="https://user-images.githubusercontent.com/15219142/102967799-67917c80-44fb-11eb-90b2-6b234ea03cdf.png">

another example

Reply:
<img width="371" alt="Screenshot 2020-12-23 at 8 47 27" src="https://user-images.githubusercontent.com/15219142/102967851-80019700-44fb-11eb-89a5-6f19d8cb150a.png">

and when file is open you can see the path

Reply:
IntelliJ have the same features but the problem is perhaps when you want to find all LandingPage related file:
![Screenshot 2020-12-23 at 10 56 59 AM](https://user-images.githubusercontent.com/7648194/102982071-9ff08500-4512-11eb-9f16-de82f8c902dc.png)
You wont find the utils.ts
But now when you search banner releated file, bannerUtils is visible:
![Screenshot 2020-12-23 at 10 57 13 AM](https://user-images.githubusercontent.com/7648194/102982153-c1517100-4512-11eb-8be7-ecabc027e75f.png)